packages/graph-retriever/src/graph_retriever/testing/adapter_tests.py

```diff
@@ -190,6 +190,50 @@ class AdjacentCase(AdapterComplianceCase):
             "horse",
         ],
     ),
+    AdjacentCase(
+        id="numeric",
```

Comment:
Add some cases. These revealed at least one bug in opensearch regarding the use of numeric values (which need to be converted to text for opensearch queries).